doc/srv6/sonic-srv6.yang

```diff
@@ -0,0 +1,79 @@
+module sonic-srv6 {
+    revision 2024-12-05 {
+		description
+			"Initial revision.";
+	}
+    namespace "http://github.com/sonic-net/sonic-srv6";
+
+    import ietf-yang-types {
+		prefix yang;
+	}
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+	import sonic-common {
+		prefix scommon;
+	}
+
+    container sonic-srv6 {
+        scommon:db-name "CONFIG_DB";
+
+        list SRV6_MY_SID_TABLE {
+            key "ip-address";
+			scommon:key-delim "|";
+			scommon:key-pattern "SRV6_MY_SID_TABLE|{ip-address}";
+
+            leaf ip-address {
+                type inet:ipv6-address;
+            }
+
+            leaf block_len {
+                type uint8 {
+					range "1..128";
+				}
+                
+                default 32;
+            }
+
+            leaf node_len {
+                type uint8 {
+					range "1..128";
+				}
+
+                default 16;
+            }
+
+            leaf func_len {
+                type uint8 {
+					range "1..128";
+				}
+
+                default 16;
+            }
+
+            leaf arg_len {
+                type uint8 {
+					range "1..128";
+				}
+
+                default 0;
+            }
+
+            leaf action {
+                type enumeration {
+                    enum uN;
```

Comment:
This means that you support only uSID not regular end.dt6, end.dt46..etc, correct, we should mention clearly in the HLD that static SRv6 supports only uSID based handling.

Reply:
I added a table for the current list of supported behaviors. Feel free to extend the list after you add the necessary support and tests.